### ISSUE TITLE
The deprecated 'TokenResponseTypeHandler' is configured as the 'ResponseTypeHandlerImplClass' for supported response types.

### ISSUE THREAD

**Issue by malithie** [wso2/product-is#2819]
The default configuration of the identity.xml includes the deprecated 'org.wso2.carbon.identity.oauth2.authz.handlers.TokenResponseTypeHandler' class being configured   for supported response types
```
<SupportedResponseTypes>
            <SupportedResponseType>
                <ResponseTypeName>token</ResponseTypeName>
       <ResponseTypeHandlerImplClass>org.wso2.carbon.identity.oauth2.authz.handlers.CodeResponseTypeHandler</ResponseTypeHandlerImplClass>
            </SupportedResponseType>
            <SupportedResponseType>
                <ResponseTypeName>id_token</ResponseTypeName>
                <ResponseTypeHandlerImplClass>org.wso2.carbon.identity.oauth2.authz.handlers.TokenResponseTypeHandler</ResponseTypeHandlerImplClass>
            </SupportedResponseType>
            <SupportedResponseType>
                <ResponseTypeName>id_token token</ResponseTypeName>
                <ResponseTypeHandlerImplClass>org.wso2.carbon.identity.oauth2.authz.handlers.TokenResponseTypeHandler</ResponseTypeHandlerImplClass>
            </SupportedResponseType>
        </SupportedResponseTypes>
```
This should be fixed configuring appropriate implementation classes that handles each response type.

**Comment by mefarazath** [comment]
Fixed with https://github.com/wso2/carbon-identity-framework/pull/1446

**Comment by mefarazath** [comment]
@pulasthi7 actually we had to revert wso2/carbon-identity-framework@fc25ba3 since the classes that replace the deprecated TokenResponseTypeHandler has some logical flaws. That's the idea behind keeping this issue open.

**Comment by pulasthi7** [comment]
@mefarazath , Ok. This should have automatically closed when 5.11.x is merged to master and that commit message result in closing the issue.
